src/tactic/command/login_group_trigger.py

```diff
@@ -46,14 +47,36 @@ def execute(my):
 
         login_group = login_group_name.lower()
         login_group = login_group.replace(" ", "_")
```

Comment:
Complete. I used the re.sub() and replaced all non-word characters with _. I put the .lower() after the re.sub() is done, and it looks like the re.sub takes care of non-ASCII characters as well and the lower works once the re.sub() is done.